string.go

```diff
@@ -747,6 +748,76 @@ func (s *String) MatchAll(re string) []Match {
 	return ret
 }
 
+// IsASCII succeeds if all string characters are ASCII
+//
+// Example:
+//
+// str := NewString(t, "Hello")
+// str.IsASCII()
+func (s *String) IsASCII() *String {
+	s.chain.enter("IsASCII()")
+	defer s.chain.leave()
+
+	if s.chain.failed() {
+		return s
+	}
+
+	isASCII := true
+	for _, c := range s.value {
+		if c > unicode.MaxASCII {
+			isASCII = false
+			break
+		}
+	}
+
+	if !isASCII {
+		s.chain.fail(AssertionFailure{
+			Type:   AssertContainsSubset,
+			Actual: &AssertionValue{s.value},
+			Errors: []error{
+				errors.New("expected: all string characters are ascii"),
+			},
+		})
```

Comment:
AssertContainsSubset requires Actual and Expect options, see comment to that constant:

```
	// Check expression: [Actual] contains subset [Expected]
	AssertContainsSubset
	AssertNotContainsSubset
```

Since here we don't have any expected value,I think we can instead use more generic AssertValid type, which requires only Actual option.

Reply:
Same for second function.

Reply:
I fixed ed4c868